docs/source/guides/libraries/java.rst

```diff
@@ -1,147 +1,38 @@
 Java Library
 ------------
 
-Prerequisites
-^^^^^^^^^^^^^
+The source code of the library, which can be used with Java and Kotlin projects can be found here: https://github.com/hyperledger/iroha-java/
 
-- Java 6
-- Gradle
+Please, read corresponding README.md file in aim to get more actual information
 
-Build Process
+Prerequisites
 ^^^^^^^^^^^^^
 
-Clone Iroha repository
-
-.. code-block:: shell
-
-  git clone https://github.com/hyperledger/iroha.git --depth=1
-  cd iroha
-
-.. note:: For the latest version checkout a ``develop`` branch by adding
-  ``-b develop`` parameter.
-
-Now we need to build our Java native library. Run
-``example/java/build_library.sh``, which will take care of it.
-
-.. code-block:: shell
-
-  cd example/java
-  ./build_library.sh
-
-.. note:: ``build_library.sh`` script creates a ``dist`` folder with
-  files, needed in :ref:`java-how-to-use` section.
+- Java 6 / Kotlin
+- Gradle
 
-.. _java-how-to-use:
 
 How to Use
 ^^^^^^^^^^
 
-There are two ways of adding the Java library to your project:
-
-1. Import Java bindings through Maven Central
-2. Compile Java bindings manually
-
-Both options are described in the following sections.
-
-Import Java Bindings from Maven Central
-"""""""""""""""""""""""""""""""""""""""
-
-First of all, you need to copy ``example/java/dist/libirohajava.jnilib`` to
-the root folder of your project.
-
-.. code-block:: shell
-
-  cp dist/libirohajava.jnilib /path_to_your_project
-
-If you use **Gradle**, add the following line to your ``build.gradle`` file:
+To use Iroha in your Java/Kotlin project you should declare a dependency in the ``build.gradle`` file:
 
 .. code-block:: groovy
-  :caption: build.gradle
-
-      compile group: 'jp.co.soramitsu', name: 'iroha', version: ‘0.0.7’
-
-    If you use **Maven**, add this to your ``pom.xml``:
 
-.. code-block:: xml
-  :caption: pom.xml
+  compile "com.github.hyperledger.iroha-java:client:$irohaJavaVersion-SNAPSHOT"
```

Comment:
Is it good practice to define SNAPSHOT version in documentation?

Reply:
Will add more detailed and correct information about versions of the lib